FootballStats/FootballStats/Data/Model/Favourite/FavouritePlayer.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  FavouritePlayer.swift
+//  FootballStats
+//
+//  Created by Hoang Linh Nguyen on 12/10/2023.
+//
+
+import Foundation
+
+struct FavouritePlayer {
```

Comment:
```suggestion
struct FavouritePlayer: Codable {
```

Các object default thêm Codable và coding key đi em